scene/main/viewport.cpp

```diff
@@ -1987,12 +1968,11 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 			}
 		}
 
-		Control *from = gui.key_focus ? gui.key_focus : nullptr; //hmm
+		Control *from = gui.key_focus ? gui.key_focus : nullptr;
 
-		//keyboard focus
-		//if (from && p_event->is_pressed() && !p_event->is_alt_pressed() && !p_event->is_meta_pressed() && !p_event->key->is_command_pressed()) {
```

Comment:
added in https://github.com/godotengine/godot/commit/72fcb8a35beab251d01864bc67da3e3a8e75aed6
6 years ago, was never commented in